### DISCUSSION TITLE
Can't connect to qBittorrent-nox 4.4.4

### DISCUSSION THREAD

**Discussion by Izumiko** [jesec/flood#589] [Q&A] [answered]
flood works fine with qbittorrent-nox 4.4.3.1, but when I upgraded to the [latest version 4.4.4](https://github.com/userdocs/qbittorrent-nox-static/releases/tag/release-4.4.4_v2.0.7), it reports "Cannot connect to the client". Has anyone ever had the same problem?

**Comment by nickexyz** [comment]
Same issue here, seems like QBT tightened the security a bit. I'm looking around to see what changed.

**Comment by quorn23** [comment]
@jesec are you aware of this? A change in 4.4.+ broke the connection with flood.

**Comment by Rubensei** [comment]
It can connect, but only during the initial setup, as soon as you try to login again (in other browser/device or after logging out) you will get the error and the logs will show an issue on `updateAuthCookie`

EDIT: Might have something to do with the PRs mentioned in [this PR](https://github.com/rmartin16/qbittorrent-api/pull/108) (?)

**Comment by dasbootbee** [comment]
Also broken on vanilla qBittorrent v4.4.5.  Suddenly getting error: Cannot connect to the client
Connection could not be verified.

**Comment by dasbootbee** [comment]
For anyone having the issue, using qBittorrent 4.4.3 (or pulling an image with that tag) allows Flood to function again.

**Reply by Izumiko** [reply]
I am temporarily using the qBittorrent's built-in WebUI and VueTorrent.

**Comment by enoch85** [comment]
Now when qBitorrent 4.5 are out, can anyone confirm that it's working?

**Reply by Izumiko** [reply]
I tried the [latest rolling build](https://github.com/jesec/flood/actions/runs/3271222407), and the bug has been fixed. According to issue #592, the docker image `jesec/flood:master` is also working.

**Reply by enoch85** [reply]
Works! https://nightly.link/jesec/flood/workflows/publish-rolling/master

**Reply by TheFatPanda-Dev** [reply]
It doesn't work for me.

**Reply by btTeddy** [reply]
> Now when qBitorrent 4.5 are out, can anyone confirm that it's working?

Nope. Still doesn't

**Reply by enoch85** [reply]
Seems like the nightlys are not possible to download anymore, Does anyone have a built package from master?

**EDIT:**
I can answer my own question, it's here: https://github.com/enoch85/flood/releases/tag/master-20230405

You're welcome!

**Comment by Syco54645** [comment]
It needs the develop branch of flood. I have installed a container just for
flood in develop and it works fine. Silly to have 2 containers just because
there has not been a release but it is what it is.

**Comment by trim21** [comment, marked as answer]
try latest release
